test/e2e/integration/frontend-test/all-process-steps.ts

```diff
@@ -70,10 +70,8 @@ function testConfirmationPage() {
 
   cy.snapshot('confirm');
 
-  // TODO: Enable this when it doesn't break custom receipt. It would be nice to make sure refreshing on a confirm step
-  // restores all state and we're not stuck in any way here.
-  // cy.reloadAndWait();
-  // cy.get(appFrontend.confirm.container).should('be.visible');
+  cy.reloadAndWait();
+  cy.get(appFrontend.confirm.container).should('be.visible');
```

Comment:
Oh, nice! :raised_hands: That should probably have been commented back in a good while ago.

Reply:
yeah probably for v4.0.0 😅